lxd/main.go

```diff
@@ -15,6 +15,7 @@ import (
 
 // Global arguments
 var argAuto = gnuflag.Bool("auto", false, "")
+var argPreseed = gnuflag.Bool("preseed", false, "")
```

Comment:
I don't believe gnuflag supports that.

I suspect the easiest will be to always have it take a filename and just support "-" as meaning /dev/stdin.